### PR TITLE
Fix pasting smart object copied from other sources (Illustrator) freezes Photoshop

### PR DESCRIPTION
**Pull request by shaoshing**
Yet another fix related to `place` command. Unfortunately, @mcilroyc's fix in #2965 using `toolModalStateChanged` does not cover this issue, due to the reason similar to place-object menu commands (https://github.com/adobe-photoshop/spaces-design/pull/2965/files#diff-0ade3704b4a758bac0c90e3fb8835095R82). 

This will address issue #2282

### CODE REVIEW

File: src/js/actions/edit.js [hunk 1]
```diff
@@ -127,17 +128,26 @@ define(function (require, exports) {
...
+        // until the object is committed.
+        return this.transfer(policyActions.suspendAllPolicies)
+            .bind(this)
+            .then(function () {
+                return this.transfer(menu.nativeModal, {
+                    commandID: PASTE_NATIVE_MENU_COMMMAND_ID
+                });
+            })
+            .finally(function () {
+                if (this.flux.store("policy").areAllSuspended) {
```
Comment (mcilroyc) [inline src/js/actions/edit.js]: isn't this a function `areAllSuspended()`

Comment (shaoshing) [inline src/js/actions/edit.js]: :see_no_evil: You are truly right!

**Comment by shaoshing** [comment]
@mcilroyc I've fixed the wrong function call. I saw another one but is fixed in your PR, so I will just leave it there to avoid conflict.

**Comment by mcilroyc** [comment]
alright, looks good!

**Pull request merged**